integration-tests/oidc-client-wiremock/src/test/java/io/quarkus/it/keycloak/OidcClientTest.java

```diff
@@ -28,6 +30,9 @@
 @QuarkusTestResource(KeycloakRealmResourceManager.class)
 public class OidcClientTest {
 
+    @InjectWireMock
```

Comment:
@Eng-Fouad This injection is not necessary, `KeycloakRealmResourceManager` is starting a Wiremock server and you have stubs created there

Reply:
Then how can I access `WireMockServer` instance? I need it to invoke `server.setScenarioState()`.

Reply:
Oh, I see you you using the scenario...